libs/h3/src/main/java/org/opensearch/geospatial/h3/H3.java

```diff
@@ -35,6 +35,7 @@
  */
 public final class H3 {
 
+    public static int MIN_H3_RES = Constants.MIN_H3_RES;
```

Comment:
Why do we need this variable?

Reply:
For consistency, we added this to H3.java similar to MAX_H3_RES.

Reply:
Can we remove MAX_H3_RES instead?

Reply:
H3 class defines the public API of the H3 library. We imported this PR. I will create an issue to refactor/clean up, for now will keep it as it is. Will that work?

Reply:
https://github.com/opensearch-project/geospatial/issues/170

Reply:
Fair enough.